go/vendor/github.com/keybase/go-crypto/openpgp/packet/private_key.go

```diff
@@ -88,8 +88,6 @@ func (pk *PrivateKey) parse(r io.Reader) (err error) {
 		return errors.UnsupportedError("deprecated s2k function in private key")
 	}
 
-
-
```

Comment:
Any reason for these formatting changes in vendored code? Or is this a version bump?